### ISSUE TITLE
[Bug] Improve accessibility of top menu (missing role attributes)

### ISSUE THREAD

**Issue by textagroup** [piwik/piwik#22188]
### What happened?

Performing a W3 validation check on the dashboard reports the following for a couple of parent unordered lists

Error: An element with role=menuitem must be contained in, or owned by, an element with role=menubar or role=menu.

This is very minor but it may affect how some screen readers process th epage.

### What should happen?

I should not see this error when running a W3 validation report

### How can this be reproduced?

Install Matomo and run a W3 validation report

### Matomo version

 5.1.0-b1

### PHP version

8.1.2

### Server operating system

Ubuntu 22.04.1 LTS

### What browsers are you seeing the problem on?

Chrome

### Computer operating system

Ubuntu 22.04.1 LTS

### Validations

- [X] Read our [Contributing Guidelines](https://github.com/matomo-org/matomo/blob/5.x-dev/CONTRIBUTING.md).
- [X] Follow our [Security Policy](https://github.com/matomo-org/matomo/blob/5.x-dev/SECURITY.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating duplicates.
- [X] The provided steps to reproduce is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the Bug.
